llvm/lib/Target/RISCV/RISCVInstrInfo.cpp

```diff
@@ -1501,6 +1525,11 @@ bool RISCVInstrInfo::isBranchOffsetInRange(unsigned BranchOp,
   switch (BranchOp) {
   default:
     llvm_unreachable("Unexpected opcode!");
+  case RISCV::NDS_BBC:
+  case RISCV::NDS_BBS:
+  case RISCV::NDS_BEQC:
+  case RISCV::NDS_BNEC:
+    return isIntN(11, BrOffset);
```

Comment:
Can you use `isInt<11>(BrOffset)`? Not sure why this pattern of using isIntN started, but we should use the template version and fix the rest.